### PR TITLE
Fixed issue #18823: STATA Export fails with certain question types 

### PR DESCRIPTION
**Pull request by gabrieljenik**
Very basic sanity test added

### CODE REVIEW

File: tests/unit/helpers/remotecontrol/ExportResponsesStataXmlTest.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    {
+        $sessionKey = $this->handler->get_session_key($this->getUsername(), $this->getPassword());
+
+        $result = $this->handler->export_responses($sessionKey, self::$surveyId, 'stataxml');
+        $this->assertNotNull($result);
+    }
+
+    public static function tearDownAfterClass(): void
+    {
+        self::$testSurvey->delete();
```
Comment (Shnoulle) [inline tests/unit/helpers/remotecontrol/ExportResponsesStataXmlTest.php:47]: If i don't make error , it's already in parent

Comment (Shnoulle) [reply tests/unit/helpers/remotecontrol/ExportResponsesStataXmlTest.php:47]: Strange : https://github.com/LimeSurvey/LimeSurvey/blob/804319af283f3be4c01c676279caeae8027d99f7/tests/TestBaseClass.php#L133

**Review by Shnoulle — APPROVED** [review]

**Comment by Shnoulle** [comment]
Just remove the 3 private var and i mrege https://sonarcloud.io/project/issues?id=LimeSurvey_LimeSurvey&pullRequest=3220&resolved=false&types=CODE_SMELL

(after check the cancelled 7.4 push)

File: tests/unit/helpers/remotecontrol/ExportResponsesStataXmlTest.php (hunk 1 above)
Comment (gabrieljenik) [reply tests/unit/helpers/remotecontrol/ExportResponsesStataXmlTest.php]: Have just updated it. Let me see how it goes now.

**Comment by gabrieljenik** [comment]
All green

**Pull request merged**